views/pins.ejs

```diff
@@ -5,53 +5,62 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="stylesheet" href="css/pins.style.css">
+    <title>CodePin</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,600;1,600&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="css/pinsStyle.css">
 </head>
 
 <body>
-    <h1>CodePins</h1>
-    <a href="/pins/dashboard">Go to public dashboard</a>
-    <div id="snippetsContainer">
+    <header>
+        <img class="logo" src="images/codepin-logo.jpg" alt="CodePin Logo">
+        <ul>
+            <li class="header" id="logoutButton"><a class="btn red logout" href="/logout" role="button">Logout</a></li>
+            <li class="header" id="toPublicBtn"><a href="/pins/dashboard" class="btn blue pin-dash" role="button">Go to
+                    public dashboard</a></li>
+        </ul>
+    </header>
+    <main>
+        <h1 class="pin-dashboard">Pin It.</h1>
+        <div class="main-container">
+            <div class="snippet out" id="snippetsContainer">
         <% snippets.forEach( el=> { %>
-            <div class='todoItem' data-id='<%=el._id%>'>
-                <h3>
+            <div class='snippet in userDash' data-id='<%=el._id%>'>
+                <h3 class="snippet">
                     <%= el.title %>
                 </h3>
-                <div class='wrapCode code'>
+                <div class="wrapCode code">
```

Comment:
This div and the same div on the dashboard view has to be all on one line to keep the formatting. 
<div class='wrapCode code"><%= decodeURIComponent(el.snippet) %></div>

Reply:
@JordanHillDev  I don't understand what you mean. Can you elaborate?